Editor/Scripts/wind_forces_tutorial.py

```diff
@@ -0,0 +1,94 @@
+        self.add_step(TutorialStep("Add a tag component for the entity",  """<html><p style="font-size:13px"> Edit the tag value to <b>global_wind</b>.
+                <br><br> There are two types of wind: global and local, used to specify which entities provide wind forces. <br><br>
+                If you choose to use the Local wind tag property, the wind force affects only certain entities.</p></html>""", "EntityOutlinerWidgetUI"))
+        self.add_step(TutorialStep("Add a PhysX collider component",  """<html><p style="font-size:13px"> Wind providers must have a PhysX collider. For local wind forces, the collider specifies the volume that contains the wind force. 
+                Objects are affected by the local wind force only when they are inside the wind provider's PhysX collider.
+                In Entity Inspector, choose Add Component and add a PhysX Collider component to the entity. <br><br> </p></html>""", "InspectorMainWindow"))
+        self.add_step(TutorialStep("Set the collider shape", """<html><p style="font-size:13px">We'll use a simple box for the volume of our local wind force.<br><br>
+                In <b>Entity Inspector</b>, in the <b>PhysX Collider</b> component, set the <b>Shape</b> property to 
+                <strong><code style="font-size:14px;color:#E44C9A">"Box"</code></strong>.<br><br>
+                The wind force can only affect entities that are within this box collider shape. </p></html>""", "InspectorMainWindow"))
+        self.add_step(TutorialStep("Resize the wind provider", """<html><p style="font-size:13px">The box is quite small. Let's enlarge it so that it affects a larger area and so that we can easily position the cloth entity inside it.
+                <br><br>In <b>Entity Inspector</b> in the <b>PhysX Collider</b> component, set the <b>X</b>, <b>Y</b>, and <b>Z</b> components of the <b>Dimensions</b> property to 
+                <strong><code style="font-size:14px;color:#E44C9A">"text"</code></strong>.</p></html>""", "InspectorMainWindow"))
+        self.add_step(TutorialStep("Position the Entity", """<html><p style="font-size:13px"> Use the <b>Move</b> tool to position the wind provider entity in the level. 
+                Ensure that the bottom of the PhysX collider box is roughly on the ground plane, and that the collider is in the camera's view frustum.</p></html>""", "InspectorMainWindow"))
+        self.add_step(TutorialStep("Add a PhysX Force Region component", """<html><p style="font-size:13px"> 
+                <br><br>Our wind provider entity has a Tag that identifies it as a local wind provider and a box collider that defines a volume for the wind force. We need to add a <b>PhysX Force Region</b> that defines the wind force.<br><br>
+                In <b>Entity Inspector</b>, choose <b>Add Component</b>, and select <b>PhysX Force Region</b> from the component list. </p></html>""", "InspectorMainWindow"))
```

Comment:
Highlight the add component button.

```suggestion
        self.add_step(TutorialStep("Add a PhysX Force Region component", 
                """<html><p style="font-size:13px">The wind provider entity has
                 a tag that identifies it as a local wind provider and a box 
                collider that defines a volume for the wind force. You need to 
                add a <b>PhysX Force Region</b> component that defines the wind 
                force.<br><br>In <b>Entity Inspector</b>, choose <b>Add 
                Component</b>, and select <b>PhysX Force Region</b> from the 
                component list.</p></html>""", "m_addComponentButton"))
```